add malformed yaml

kirbyquerby opened the pull request.

odeke-em commented on .orijtech/bencher.yaml:
```diff
@@ -0,0 +1 @@
+haha im malformed what're you gonna do about it
```
Wrong file name .bencher/config.yaml is what you want.

kirbyquerby replied:
Haha yeah, and to think I wrote documentation for this just yesterday 🙃

kirbyquerby committed:
add malformed yaml


Also change a .go file :) 


it would help if I gave the config file the right name...
